src/timeMachine/components/TimeMachineFluxEditor.tsx

```diff
@@ -74,37 +61,31 @@ const TMFluxEditor: FC<{variables: Variable[]}> = props => {
     [activeQueryText, handleSetActiveQueryText]
   )
 
-  return useMemo(() => {
```

Comment:
Memoization was put in to reduce re-rendering when filtering/searching on flux functions.
Note to other reviewers --> confirmed that it's no longer needed. All clear.